packages/appstore/src/components/modals/cashier-wallet-modal/provider.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
+import AccountTransfer from '@deriv/cashier/src/pages/account-transfer';
+
+export type TWalletType = 'real' | 'demo' | 'p2p' | 'payment_agent';
+
+export const getCashierOptions = (type: TWalletType) => {
+    switch (type) {
+        case 'real':
+            return [
+                {
+                    icon: 'IcAdd',
+                    label: localize('Deposit'),
+                    //Remove AccountTransfer after QA testing (testing scroll behaviour)
+                    content: isMobile() ? (
+                        <>
+                            <AccountTransfer onClose={() => {}} />
+                            <AccountTransfer onClose={() => {}} />
+                        </>
+                    ) : (
+                        <p>Deposit Real</p>
+                    ),
+                },
+                { icon: 'IcMinus', label: localize('Withdraw'), content: <p>Withdraw Real</p> },
+                {
+                    icon: 'IcAccountTransfer',
+                    label: localize('Transfer'),
+                    content: <p>Transfer Real</p>,
+                },
+                {
+                    icon: 'IcStatement',
+                    label: localize('Transactions'),
+                    content: <p>Transactions Real</p>,
```

Comment:
add localize

Reply:
@mahdiyeh-deriv no need to add localize here, there is a mocked content